addons/delivery/wizard/choose_delivery_package.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class ChooseDeliveryPackage(models.TransientModel):
+    _name = 'choose.delivery.package'
+    _description = 'Choose delivery package type when there is a carrier on the picking'
```

Comment:
`_description` will be the name of the model in the Settings > Technical > Models list view, I suggest something more name-like ('Delivery Package Selection Wizard' or some other crap of the same kind)